code/game/objects/random/mob/spider.dm

```diff
@@ -9,6 +9,7 @@
 				/mob/living/carbon/superior_animal/giant_spider/nurse/midwife = 15,\
 				/mob/living/carbon/superior_animal/giant_spider/nurse/queen = 5,\
 				/mob/living/carbon/superior_animal/giant_spider/nurse/recluse = 4,\
+				/mob/living/carbon/superior_animal/giant_spider/plasma = 4,\
```

Comment:
```suggestion
				/mob/living/carbon/superior_animal/giant_spider/plasma = 2,\
```
these should be rarer then recluse as they brake pipes/walls and AOE their own making them a bad breed

Reply:
It gives the guild something to repair.